sos/plugins/logs.py

```diff
@@ -50,83 +40,31 @@ def setup(self):
             if os.path.isfile(i):
                 self.add_copy_spec(i)
 
-        if self.get_option('all_logs'):
-            self.add_journal(boot="this", since=self.get_option("since"),
-                             allfields=True, output="verbose")
-            self.add_journal(boot="last", allfields=True, output="verbose")
```

Comment:
RH engineering sometimes needs the verbose journals. We moved this to an optional collection in #1225 as a result of requests to not make verbose journals a default collection, but I don't agree with removing the ability to collect them entirely.

Reply:
Before my change, this has "this" boot defined with since, but not last.  So we could end up collecting a full last boot and only a partial this boot.

Reply:
The `since` applied just to `this` boot but not `last` is ilogical, indeed. But collecting the verbose logs makes sense. And at least on RH distros, `journalctl --boot -1 --since <yesterday>` collects nothing when boot happened 2 days ago - so if the @BryanQuigley concern is the `since` inconsistency, let add it to the `last` boot as well.

Reply:
I added both catalog and verbose to a RH only bit towards the end.  Verbose to be collected separately when all-logs specified. 

@TurboTurtle mentioned that Catalog was the really useful one for last/this boots (my reading doesn't show it's useful at all, but heh).  I left those Catalog last/this out of using since because if both occurred > since you won't get any actual boot data from either of them.

Reply:
I cant find those commands any where, in the current commit version.